opentelemetry-otlp/tests/integration_test/tests/logs.rs

```diff
@@ -84,15 +98,44 @@ mod logtests {
 
         Ok(())
     }
+
+    #[test]
+    #[cfg(any(feature = "tonic-client", feature = "reqwest-blocking-client"))]
```

Comment:
@cijothomas here's the new test. I took advantage of the temporary tokio runtime to setup the collector too.